src/test/java/teammates/test/pageobjects/InstructorEditStudentFeedbackPage.java

```diff
@@ -0,0 +1,31 @@
+package teammates.test.pageobjects;
+
+import java.io.IOException;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class InstructorEditStudentFeedbackPage extends FeedbackSubmitPage {
+
+    public InstructorEditStudentFeedbackPage(Browser browser) {
+        super(browser);
+    }
+    
+    public void clickModerationHintButton() {
+        click(By.id("moderationHintButton"));
```

Comment:
This can be a class variable instead, using the `@FindBy` annotation; especially since this same element is used in another method.